src/tactic/core.lean

```diff
@@ -697,6 +697,26 @@ meta def context_upto_hyp_has_local_def (h : expr) : tactic bool := do
   let ctx := ctx.take_while (≠ h),
   ctx.many (succeeds ∘ local_def_value)
 
+/--
```

Comment:
As below, I'd like to explain what `subst'` does in its own docstring. (`tactic.subst` doesn't even have a docstring!)
```suggestion
/--
If the expression `h` is a local variable with type `x = t` or `t = x`, where `x` is a local constant,
`tactic.subst' h` substitutes `x` by `t` everywhere in the main goal and then clears `h`.

```